{
	"id": "oval:org.debian:def:41093336860194580418285912540096455895",
	"class": "vulnerability",
	"metadata": {
		"title": "CVE-2024-3651 python-idna",
		"affected": {
			"family": "unix",
			"platform": "Debian GNU/Linux 10",
			"product": "python-idna"
		},
		"reference": [
			{
				"source": "CVE",
				"ref_id": "CVE-2024-3651",
				"ref_url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-3651"
			}
		],
		"debian": {}
	},
	"criteria": {
		"operator": "AND",
		"criterias": [
			{
				"operator": "OR",
				"criterias": [
					{
						"operator": "AND",
						"criterions": [
							{
								"test_ref": "oval:org.debian.oval:tst:2",
								"comment": "all architecture"
							},
							{
								"test_ref": "oval:org.debian.oval:tst:28828",
								"comment": "python-idna DPKG is earlier than 2.6-1+deb10u1"
							}
						]
					}
				]
			}
		],
		"criterions": [
			{
				"test_ref": "oval:org.debian.oval:tst:1",
				"comment": "Debian 10 is installed"
			}
		]
	}
}
